src/runtime/lpython_builtin.py

```diff
@@ -2,29 +2,22 @@
 #from sys import exit
 
 
-def ord(s: str) -> i32: # currently supports characters with unicode value between 32 to 126
+def ord(s: str) -> i32:
     """
     Returns an integer representing the Unicode code
     point of a given unicode character. This is the inverse of `chr()`.
     """
     if len(s) != 1:
         return -1 # not a character
-    i: i32
-    for i in range(32, 127):
-        if chr(i) == s:
-            return i
+    return i32(int(s))
```

Comment:
Good one.

Reply:
Since this does not work in CPython, it's a bug in LPython that we should fix.